tests/pkg_openwsn_sctimer/Makefile

```diff
@@ -0,0 +1,16 @@
+APPLICATION = openwsn_sctimer
```

Comment:
this forces a name clash with the actual openwsn module. I think the default should be ok (removing that line would default to `test_openwsn_sctimer`).

Reply:
This is catched by the static tests :)

Reply:
Doesn't apply